src/client/components/filter-tile/filter-tile.tsx

```diff
@@ -274,54 +277,51 @@ export class FilterTile extends React.Component<FilterTileProps, FilterTileState
     this.setState({ dragPosition: null });
   }
 
+  draggingDimension(): Dimension {
+    const { essence: { dataCube } } = this.props;
+    if (DragManager.isDraggingSplit()) {
```

Comment:
Again, this logic should be fully encapsulated in DragManager, it is responsible for returning dragging dimension (if any).

Reply:
This is possible, but we will get three variants of this, for each TileMenu. Sometimes DragManager will search dimension from it's split, sometimes from it's filter.